modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java

```diff
@@ -141,17 +131,20 @@ public void testSearch() throws Exception {
 	@Rule
 	public FrutillaRule frutillaRule = new FrutillaRule();
 
-	private static Company _company;
-	private static Indexer<CommercePriceList> _indexer;
-
 	@Inject
-	private static IndexerRegistry _indexerRegistry;
+	private CommercePriceListLocalService _commercePriceListLocalService;
 
-	private static User _user;
+	private List<CommercePriceList> _commercePriceLists;
```

Comment:
Basically just a SF thing, i think _commercePriceLists did not need to become private here and _indexer/_indexerRegistry could have stayed static in the setUpClass method.